classes/MyTableLister.php

```diff
@@ -217,8 +217,8 @@ public function selectSQL($columns, &$vars)
         foreach ($columns as $key => $value) {
             if (isset($this->fields[$key]['foreign_table']) && $this->fields[$key]['foreign_table']) {
                 $result['join'] .= ' LEFT JOIN ' . $this->fields[$key]['foreign_table']
-                    . ' ON ' . $this->table . '.' . $key
-                    . '=' . $this->fields[$key]['foreign_table'] . '.' . $this->fields[$key]['foreign_column'];
+                    . ' ON ' . '`' .$this->table . '`.`' . $key . '`'
+                    . '=' . '`' .$this->fields[$key]['foreign_table'] . '`.`' . $this->fields[$key]['foreign_column'] . '`';
```

Comment:
Ano, tady je to potřeba ošetřit, ale než pouze obalením backticks (nebo jak se tomu říká &amp;#96;), tak zavoláním $this->escapeDbIdentifier()
viz koment níže pro více info

Reply:
upravil jsem to